Vagrantfile

```diff
@@ -14,17 +15,39 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo apt-get update
-    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd tree
+    sudo apt-get install -y autoconf automake build-essential expect git \
+        libfreetype6-dev libpng12-dev libtool nmap openjdk-7-jdk \
+        openjdk-7-jre pkg-config python-all-dev python-matplotlib \
+        python-nose python-numpy python-pip python-zmq sshpass trace-cmd \
+        tree wget
     sudo pip install ipython[notebook] pandas
     sudo apt-get remove -y w3m
 
     ln -s /vagrant /home/vagrant/lisa
+
+    cd /home/vagrant/lisa
+    ANDROID_SDK_URL="https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz"
+    if [ ! -e ./tools/android-sdk-linux ]; then
+        echo "Downloading Android SDK [$ANDROID_SDK_URL]..."
+        wget -qO- $ANDROID_SDK_URL | tar xz -C tools
+        expect -c '
+            set timeout -1;
+            spawn ./tools/android-sdk-linux/tools/android update sdk --no-ui \
+                -t tool,platform-tool,platform;
+            expect {
+                "Do you accept the license" { exp_send "y\r" ; exp_continue }
+                eof
+            }
+        '
+    fi
+
     chown vagrant.vagrant /home/vagrant/lisa
     echo cd /home/vagrant/lisa >> /home/vagrant/.bashrc
     for LC in $(locale | cut -d= -f1);
     do
         echo unset $LC  >> /home/vagrant/.bashrc
     done
+    echo "export ANDROID_HOME=/vagrant/tools/android-sdk-linux" >> /home/vagrant/.bashrc
     echo source init_env >> /home/vagrant/.bashrc
   SHELL
```

Comment:
Before returning from the provisioning script, let's notice the user that the VM is ready and she can access and use it by running the
```bash
vagrant ssh
```

And notice that if you exit the SSH the machine is still running and this command
```bash
vagrant suspend
```
to actually shout down the machine when not required.